_docs-sources/guides/stay-up-to-date/cis/cis-1.5.0/deployment-walkthrough/step-5-check-your-live-infrastructure-is-cis-v1.5-compliant.md

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_label: Check your live infrastructure is CIS v1.5 compliant
+---
+
+# Step 5: Check your live infrastructure is CIS v1.5 compliant
```

Comment:
What about manual steps like Macie?

Reply:
Added a mention to the manual steps page in there and in the `core-concepts.md`